.github/CODEOWNERS

```diff
@@ -1,8 +0,0 @@
-# This is a comment.
```

Comment:
do we want DevX in here?

Reply:
Looks to me like this was the previous iteration of the `DevX` team - our team on github is still @puppetlabs/tooling which is set [here](https://github.com/puppetlabs/puppet-vscode/blob/main/CODEOWNERS)

Reply:
ahhh great